src/dao.cpp

```diff
@@ -65,16 +65,17 @@ namespace hypha
 
    void dao::propose(const name &proposer,
                      const name &proposal_type,
+                     bool publish,
                      ContentGroups &content_groups)
    {
       TRACE_FUNCTION()
       EOS_CHECK(!isPaused(), "Contract is paused for maintenance. Please try again later.");
 
       std::unique_ptr<Proposal> proposal = std::unique_ptr<Proposal>(ProposalFactory::Factory(*this, proposal_type));
-      proposal->propose(proposer, content_groups);
+      proposal->propose(proposer, content_groups, publish);
    }
 
-   void dao::vote(const name &voter, const checksum256 &proposal_hash, string &vote, string notes)
+   void dao::vote(const name& voter, const checksum256 &proposal_hash, string &vote, const std::optional<string> &notes)
```

Comment:
Is there a benefit to having notes optional?

Reply:
As this was a new param, i preffered to have it optional. I forgot to add the std::optional initially